generated/proto/encodingpb/string.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option go_package = "github.com/xichen2020/eventdb/generated/proto/encodingpb";
+
+import "github.com/xichen2020/eventdb/generated/proto/encodingpb/types.proto";
+
+package encodingpb;
+
+message StringMeta {
+    EncodingType encoding = 1;
+    CompressionType compression = 2;
+    // Byte packing for length in bits or dict encoded id.
+    int64 bytes_per_record = 3;
```

Comment:
Should be comments by `Byte packing for length in bytes...`?

Also perhaps `num_bytes_per_value` is a better name given that we don't really have a concept of record elsewhere in the codebase?

Reply:
👍